### DISCUSSION TITLE
Release 1.2.8

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#8166] [General]
## borgbackup 1.2.8 release

This release includes a simplified upgrade procedure at the top of the change log and other fixes:

https://github.com/borgbackup/borg/blob/1.2.8/docs/changes.rst#pre-125-archives-spoofing-vulnerability-cve-2023-36811

Long changelog:

https://github.com/borgbackup/borg/blob/1.2.8/docs/changes.rst#version-128-2024-03-29

Short borg 1.2 overview (from a borg 1.1 perspective):

https://www.borgbackup.org/releases/borg-1.2.html

### Installation

If you use pip to install this, use: pip install pkgconfig ; pip install "borgbackup==1.2.8"

For other installation methods and more details, please see: https://borgbackup.org/

See also the 00_README.txt file in the assets list below for a description of the available "fat binary" downloads.

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/1.2.8'>Release 1.2.8</a>.</em>

**Comment by enkore** [comment]
Worth pointing out that none of the Linux binaries contain liblzma 5.6

**Reply by ThomasWaldmann** [reply]
Yeah, they were all built using stable Debian releases (see 00_README.txt).

**Comment by Anachron** [comment]
I'm just leaving a comment here to say
1.) Thank you! Borg backup is amazing and I love it. It is by far the most delicate backup software I've used. 
2.) This release has been tested on 4+ devices (x86_64, aarch64) and it's been a breeze. 

@ThomasWaldmann you're awesome, keep it up!
